lib/reckon/ledger_parser.rb

```diff
@@ -23,7 +23,7 @@ def parse(ledger)
           date = $1
           desc = $3
           accounts = []
-        elsif date && entry =~ /^\s+([a-z\s:_\-]+)(\s*$|(\s+[\$\.,\-\d\+]+)($|\s+($|[^\$\.,\-\d\+])))/i
+        elsif date && entry =~ /^\s+([a-z\s:_\-]+)(\s*$|(\s+\-?\+?.*[\.,\-\d\+]+)(.*$))/i
```

Comment:
`.*` here seems like it'd be too generous.  I think we need some specs on this to make sure it doesn't break other stuff.